overlays/holo-nixpkgs/hpos-reset/default.nix

```
@@ -0,0 +1,22 @@
+{ stdenv, lib, makeWrapper, gitignoreSource }:
+
+with lib;
+
+{
+
+  hpos-reset = stdenv.mkDerivation rec {
+    name = "hpos-reset";
+    src = gitignoreSource ./.;
+
+    nativeBuildInputs = [ makeWrapper ];
+
+    installPhase = ''
+      install -Dm 755 hpos-reset.sh $out/bin/${name}
```

755 means `rwxr-xr-x`, right? (User `root` can read, write, execute. Group `root` can read and execute. Other users can read and execute)

From the documentation of `install`, it looks like that's the default
```
  -m, --mode=MODE     set permission mode (as in chmod), instead of rwxr-xr-x
```
So why not just remove `-m 755`?
```suggestion
      install -D hpos-reset.sh $out/bin/${name}
```

---

we have this in the update script too so I didn't want to tinker around with it. I can give it a go?

---

It's fine to leave it. I'll resolve this comment
> I can give it a go?
Not sure I understand this question